src/components/data/LocalVariablesContent.js

```diff
@@ -47,25 +65,36 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
     setEditIndex()
     setEditKey()
     setEditValue()
+    setEditDescription()
     setEditType()
   }
 
+  const toDescriptionKey = k => k + DESCRIPTION_SUFFIX
+
   const initialAttributes = _.flow(
     _.toPairs,
-    _.remove(([key]) => /^description$|:|^referenceData_/.test(key))
+    _.remove(([key]) => /^description$|:|^referenceData_/.test(key)),
+    _.map(renameAttribute),
+    _.groupBy('key'),
+    _.values,
+    _.map(getDisplayedAttribute)
   )(attributes)
```

Comment:
Now that this flow is so long, would you consider extracting this into `convertInitialAttributes` outside of the component and writing a test for it? Partially to make sure it doesn't get messed up in the future, and partly to give some concrete documentation of how it's expected to work?

Reply:
Done.